Mathlib/RepresentationTheory/Action/Limits.lean

```diff
@@ -0,0 +1,208 @@
+/-
+Copyright (c) 2020 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+import Mathlib.CategoryTheory.Abelian.FunctorCategory
+import Mathlib.CategoryTheory.Abelian.Transfer
+import Mathlib.CategoryTheory.Linear.LinearFunctor
+import Mathlib.CategoryTheory.Linear.FunctorCategory
+import Mathlib.CategoryTheory.Preadditive.AdditiveFunctor
+import Mathlib.RepresentationTheory.Action.Basic
+
+/-!
```

Comment:
Can you please add a title for this file, like
```suggestion
/-!
# Categorical properties of `Action V G`
```
or something like that...

Reply:
See the latest commit.